src/stores/FileBrowserStore/FileBrowserStore.ts

```diff
@@ -536,15 +536,17 @@ export class FileBrowserStore {
 
     @action addExportRegionIndex = (regionIndex: number) => {
         if (!this.exportRegionIndexes.includes(regionIndex)) {
-            this.exportRegionIndexes.push(regionIndex);
-            this.exportRegionIndexes.sort();
+            this.exportRegionIndexes = [...this.exportRegionIndexes, regionIndex];
+            this.exportRegionIndexes = [...this.exportRegionIndexes].sort();
```

Comment:
1. I think we only need to copy the array once within the action function.
2. Should `sort()` be changed to `sort((a, b) => a - b)`?